tests/tck/features/optimizer/IndexScanRule.feature

```diff
@@ -20,25 +20,25 @@ Feature: Match index selection
       | ("Tony Parker" :player{age: 36, name: "Tony Parker"})     |
       | ("Vince Carter" :player{age: 42, name: "Vince Carter"})   |
     And the execution plan should be:
-      | id | name        | dependencies | operator info                                                                                                          |
-      | 10 | Project     | 13           |                                                                                                                        |
-      | 13 | Filter      | 7            |                                                                                                                        |
-      | 7  | Project     | 6            |                                                                                                                        |
-      | 6  | Project     | 5            |                                                                                                                        |
-      | 5  | Filter      | 15           |                                                                                                                        |
-      | 15 | GetVertices | 11           |                                                                                                                        |
-      | 11 | IndexScan   | 0            | {"indexCtx": {"columnHints":{"scanType":"RANGE","column":"name","beginValue":"\"Tim Duncan","endValue":"\"Yao Ming"}}} |
-      | 0  | Start       |              |                                                                                                                        |
+      | id | name        | dependencies | operator info                                                                                                              |
+      | 10 | Project     | 13           |                                                                                                                            |
```

Comment:
Add the following case and profile the `begin_value` and `end_value` in `ColumnHints`
```
profile match (v:player) where v.age>30 and v.age<=40 return v
```

Reply:
OK